docs/docs/education-licensing.md

```
@@ -0,0 +1,21 @@
+# Licensing quick reference
+
+You do not need to become an expert in licensing to start. You need to know enough to make a decision and not accidentally restrict your work or someone else's.
+
+### For content (lecture notes, course material, research papers)
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)